docs/en/latest/admin-api.md

```diff
@@ -21,13 +29,41 @@ title: Admin API
 #
 -->
 
+## Description
+
 The Admin API lets users control their deployed Apache APISIX instance. The [architecture design](./architecture-design/apisix.md) gives an idea about how everything fits together.
 
-By default, the Admin API listens to port `9180` when APISIX is launched. This can be changed by modifying your configuration file ([conf/config.yaml](https://github.com/apache/apisix/blob/master/conf/config.yaml)).
+## Configuration
+
+When APISIX is started, the Admin API will listen on port `9180` by default and take the API prefixed with `/apisix/admin`.
+
+Therefore, to avoid conflicts between your designed API and `/apisix/admin`, you can modify the configuration file [`/conf/config.yaml`](https://github.com/apache/apisix/blob/master/ conf/config.yaml) to modify the default listening port.
```

Comment:
```suggestion
Therefore, to avoid conflicts between your designed API and `/apisix/admin`, you can modify the configuration file [`/conf/config.yaml`](https://github.com/apache/apisix/blob/master/conf/config.yaml) to modify the default listening port.
```